posthog/tasks/process_event.py

```diff
@@ -67,6 +67,8 @@ def _alias(previous_distinct_id: str, distinct_id: str, team_id: int, retry_if_f
 def store_names_and_properties(team: Team, event: str, properties: Dict) -> None:
     # In _capture we only prefetch a couple of fields in Team to avoid fetching too much data
     save = False
+    if not isinstance(event, str) and not isinstance(event, bytes):
+        raise TypeError("Event must be a string")
```

Comment:
This won't be returned to the user submitting the event, it'll just show up in Sentry. If I think what I'd want for debugging purposes it'd be better if we just convert objects to strings (max 200 characters) so that I can at least see them come in and fix whatever happens, rather than have a silent failure.

Reply:
sure, updated!